scripts/validate_rpm_specs.py

```diff
@@ -89,4 +100,5 @@ def parse_cli_options():
         sys.exit(1)
     args = parse_cli_options()
     if not validate_rpm_specs(args.rpm_specs_base_dir):
+        LOG.info("Packages not valid.")
```

Comment:
I'd log a more precise message, like "RPM spec files contain errors."
A similar message could be logged in the YAML validation error.